incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -470,81 +489,89 @@ would require them to set the `allowCascadingDeletion` property of the child
 namespace.
 
 <a name="admin-types"/>
```

Comment:
Do we need to remove the old anchor or you feel safer to leave it for now?

Reply:
Leave it indefinitely since there might be other places that link to this anchor.

Reply:
sg